codecarbon/emissions_tracker.py

```diff
@@ -41,34 +41,48 @@ def __init__(
         self,
         project_name: str = "codecarbon",
         measure_power_secs: int = 15,
+        measure_occurrence_before_calling_api: int = 2,
```

Comment:
I get the point but very lengthy var name... :)

Reply:
`api_call_lag` ?

Reply:
`api_call_interval`

Reply:
Done